en-US/firefox/retention/thank-you.lang

```diff
@@ -0,0 +1,113 @@
+## NOTE: this file covers 2 pages: https://www-dev.allizom.org/etc/firefox/retention/thank-you-a/ and https://www-dev.allizom.org/etc/firefox/retention/thank-you-b/
+## URL: https://www-dev.allizom.org/%LOCALE%/etc/firefox/retention/thank-you-a/
+
+# HTML Title
+;Thank You
+Thank You
+
+
+# HTML Description
+;Thank you page.
+Thank you page.
+
+
+;It’s all thanks to you
+It’s all thanks to you
+
+
+# Used in thank-you-a version https://www-dev.allizom.org/etc/firefox/retention/thank-you-a/
+;Choosing Firefox helps Mozilla make the Internet a better place. Here’s what you can do next:
+Choosing Firefox helps Mozilla make the Internet a better place. Here’s what you can do next:
+
+
+;Make Firefox your default browser
+Make Firefox your default browser
+
+
+# Action (setting Firefox as default) takes only 1 minute
+;1 min action
+1 min action
+
+
+;Set as your default
+Set as your default
+
+
+;Get Firefox on your phone
+Get Firefox on your phone
+
+
+# Action (installing Firefox on your phone) takes only 2 minutes
+;2 min install
+2 min install
+
+
+;Download Firefox
+Download Firefox
+
+
+;Tell your friends about Firefox
+Tell your friends about Firefox
+
+
+# Action (sharing a link to Firefox) takes only 1 minute
+;1 min share
+1 min share
+
+
+;Send a tweet
+Send a tweet
+
+
+# Tweet: keep it short, since emojis and URL are used as well
```

Comment:
After playing with the page, I think we can use a LENGTH_LIMIT=107

Reply:
Thanks, let's try with an explicit length.